README.md

```diff
@@ -47,6 +47,22 @@ Options:
 
 To create a new Access Token, navigate to [Personal Access Tokens](https://github.com/settings/tokens) and create a click "Generate new token."
 
+### Data source with provisioning
+[It’s now possible to configure data sources using config files with Grafana’s provisioning system.](https://grafana.com/docs/grafana/latest/datasources/elasticsearch/#configure-the-data-source-with-provisioning)
```

Comment:
Can you please include this as well?

Reply:
@valentinsavenko

Reply:
This yaml-part? It's already in there this would double it?

Reply:
We usually add provisioning in the format above thats why I thought that it would make sense to include it that way. But I guess people can figure it out from the prom operator yml whats needed.